projects/demo-maps/src/app/route-components/route-example-owc-layers/route-example-owc-layers.component.ts

```diff
@@ -0,0 +1,75 @@
+import { HttpClient } from '@angular/common/http';
+import { Component, HostBinding, OnInit } from '@angular/core';
+import { OsmTileLayer } from '@dlr-eoc/base-layers-raster';
+import { IMapControls, MapOlService } from '@dlr-eoc/map-ol';
+import { LayerGroup, LayersService } from '@dlr-eoc/services-layers';
+import { MapStateService } from '@dlr-eoc/services-map-state';
+import { IEocOwsContext, OwcJsonService } from '@dlr-eoc/services-ogc';
+import { first } from 'rxjs/operators';
+import { ZommNumberControl } from '../route-example-layers/ol-custom-control';
+
+@Component({
+  selector: 'app-route-example-owc-layers',
+  templateUrl: './route-example-owc-layers.component.html',
+  styleUrls: ['./route-example-owc-layers.component.scss'],
+  /** use different instances of the services only for testing with different routes  */
+  providers: [LayersService, MapStateService, MapOlService]
+})
+export class RouteExampleOwcLayersComponent implements OnInit {
+  @HostBinding('class') class = 'content-container';
+  controls: IMapControls;
+
+  constructor(
+    public layersSvc: LayersService,
+    public mapStateSvc: MapStateService,
+    public mapSvc: MapOlService,
+    private owcSvc: OwcJsonService,
+    private http: HttpClient) {
+
+    this.controls = {
+      attribution: true,
+      scaleLine: true
+    };
+  }
+
+  ngOnInit(): void {
+    this.mapSvc.map.addControl(new ZommNumberControl());
+    this.getOwcContext('assets/data/owc/ows-json-context.json');
+  }
+
+  getOwcContext(url: string) {
+    return this.http.get(url).subscribe((context: IEocOwsContext) => {
+      this.addLayersFromContext(context);
+    });
+  }
+
+  addLayersFromContext(context: IEocOwsContext) {
+    this.layersSvc.removeLayers(l => {
+      return l.id !== 'osm' && l.id !== 'ContextSelectionLayer';
+    });
+    this.owcSvc.getLayers(context, this.mapSvc.EPSG).pipe(first()).subscribe((layers) => {
+      for (const layer of layers) {
+        if (layer instanceof LayerGroup) {
+          this.layersSvc.addLayerGroup(layer, layer.filtertype);
+        } else {
+          this.layersSvc.addLayer(layer, layer.filtertype);
+        }
+      }
+    });
+
+    if (context.bbox) {
+      this.mapStateSvc.setExtent(context.bbox);
+    }
+  }
+
+
+  async downloadOwc() {
```

Comment:
This is the example how to replace @dlr-eoc/owc-control #77
The pased layers should be from `layersSvc.getLayerGroups()` so we can restore the folder property on `IOwsResource`. If we only use `layersSvc.getBaseLayers()`, `layersSvc.getOverlays()` and `layersSvc.getLayers()` as before in the component, we do not have groups so we can not create folders.